android-kotlin/app/src/main/java/com/mongodb/app/ItemAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package com.mongodb.app
+
+import android.util.Log
+import android.view.*
+import android.widget.PopupMenu
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import io.realm.OrderedRealmCollection
+import io.realm.Realm
+import io.realm.RealmRecyclerViewAdapter
+import io.realm.kotlin.where
+import io.realm.mongodb.sync.SyncConfiguration
+import org.bson.types.ObjectId
+
+/**
+ * ItemAdapter: extends the Realm-provided RealmRecyclerViewAdapter to provide
```

Comment:
No need to introduce in the comment.
```suggestion
 * Extends the Realm-provided RealmRecyclerViewAdapter to provide
```